backends/backends.c

```diff
@@ -1193,44 +1373,7 @@ void *backends_main(void *ptr) {
     }
 
 cleanup:
-#if HAVE_KINESIS
-    if(do_kinesis) {
-        kinesis_shutdown();
-        freez(kinesis_auth_key_id);
-        freez(kinesis_secure_key);
-        freez(kinesis_stream_name);
-    }
-#endif
-
-#if ENABLE_PROMETHEUS_REMOTE_WRITE
-    buffer_free(http_request_header);
-    if(do_prometheus_remote_write)
-        protocol_buffers_shutdown();
-#endif
-
-#if HAVE_MONGOC
-    if(do_mongodb) {
-        mongodb_cleanup();
-        freez(mongodb_uri);
-        freez(mongodb_database);
-        freez(mongodb_collection);
-    }
-#endif
-
-    if(sock != -1)
-        close(sock);
-
-    buffer_free(b);
-    buffer_free(response);
-
-#ifdef ENABLE_HTTPS
-    if(netdata_opentsdb_ctx) {
-        if(opentsdb_ssl.conn) {
-            SSL_free(opentsdb_ssl.conn);
-        }
-    }
-#endif
-
     netdata_thread_cleanup_pop(1);
+
     return NULL;
 }
```

Comment:
Please, add a blank line at the end of this file.

Reply:
Hm... It is already in place.

Reply:
I did not do download of the code yesterday, but at least here in Github we do not have an additional line.

Reply:
If there is no newline at the end of a file, Github shows a [little red symbol](https://octicons.github.com/icon/no-newline/).

Reply:
I did not know this, thanks!